lib/peep.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+
+class Peep 
+  
+  attr_reader :id, :peep
+
+  def initialize(id:, peep:)
+    @id = id
+    @peep = peep
+  end
+
+  def self.all 
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end
+    
+    result = connection.exec('SELECT * FROM peeps')
+    result.map do |peep| 
+      Peep.new(id: peep['id'], peep: peep['peep']) 
+    end  
+  end
+
+  def self.create(peep:)
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
you could refactor this if/else statement and use it as a def. so your code becomes DRY